src/components/molecules/nav/navigation.tsx

```diff
@@ -1,51 +1,48 @@
-/* eslint-disable */
-// This file was converted to .tsx without actually implementing typescript
-// @TODO: update this file to tsx and enable eslint
-import React, { Component } from 'react'
+import React, { Component, ReactNode } from 'react'
 import { Link } from 'gatsby'
 
 import styles from './navigation.module.css'
 
-import Menu from '../../../img/menu.inline.svg'
-import Close from '../../../img/close.inline.svg'
-import Down from '../../../img/down.inline.svg'
+import { Menu } from '../../../img/menu.inline'
+import { Close } from '../../../img/close.inline'
+// import Down from '../../../img/down.inline.svg'
```

Comment:
Do we want to keep this commented out line?

Reply:
No, I'll go ahead and delete it. There was apparently some work-in-progress to support multi-level menu, but for now, at least, it's just going to be a single level, so I ripped out the other parts. I'll remove this too

Reply:
Done